src/Components/Notifications/__tests__/PartnerShowOpenedNotification.jest.tsx

```diff
@@ -48,9 +48,7 @@ describe("PartnerShowOpenedNotification", () => {
     // show
 
     expect(screen.getByText("THEO at ART OnO")).toBeInTheDocument()
-    expect(
-      screen.getByText("April 19 – 21, 2024 — Opening Soon")
-    ).toBeInTheDocument()
+    expect(screen.getByText("April 19 – 21, 2022 — Closed")).toBeInTheDocument()
```

Comment:
Updated the year, the show should now always display "Closed" unless we jump back in time.